### DISCUSSION TITLE
I can't use my education benefits for replit

### DISCUSSION THREAD

**Discussion by notsilverhoft** [community/community#35784] [GitHub Education] [answered]
I am currently part of the GitHub education program, I signed up on Saturday and got accepted, I am a student and I heard about the benefits I could get on replit as a student, and I am creating a website that can help other student learn about technology, so I thought to sign u. The account I logged in with, linked to a different replit account, I have unlinked the old one, but after linking my main replit account, I cant get benefits any more. If any one has any ideas I would greatly appreciate it.

**Comment by jfullstackdev** [comment]
Replit has really the free version even without GitHub

**Reply by notsilverhoft** [reply]
Thank you for trying to help, I know that there is a free version, I have been using it for about 6 months, the thing is my website is running out of storage, as I am not running it as an html 5 repl, I am porting a wordpress installation on a php server, so I can run wp plugins. This within itself takes about half the storage, but I am also trying to use a flash player emulator, which takes up more than the 25% storage I have left. (The otheer 25% is the website itself.)

**Comment by notsilverhoft** [comment, marked as answer]
This has been fixed.
